src/main/java/uk/gov/hmcts/divorce/common/model/access/Applicant2Access.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.divorce.common.model.access;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.SetMultimap;
+import uk.gov.hmcts.ccd.sdk.api.HasAccessControl;
+import uk.gov.hmcts.ccd.sdk.api.HasRole;
+import uk.gov.hmcts.ccd.sdk.api.Permission;
+
+import static uk.gov.hmcts.divorce.common.model.UserRole.APPLICANT_2;
+import static uk.gov.hmcts.divorce.common.model.UserRole.APPLICANT_2_SOLICITOR;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_COURTADMIN_CTSC;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_COURTADMIN_RDU;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_LEGAL_ADVISOR;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_SYSTEMUPDATE;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CREATOR;
+
+public class Applicant2Access implements HasAccessControl {
+    @Override
+    public SetMultimap<HasRole, Permission> getGrants() {
+        SetMultimap<HasRole, Permission> grants = HashMultimap.create();
+        grants.putAll(CASEWORKER_SYSTEMUPDATE, Permissions.CREATE_READ_UPDATE);
```

Comment:
should this be only read or this is used by bulkscan and hence need CRD ?

Reply:
I believe it can be R only. Just testing locally to make sure first - then will push the change